test/uk/gov/hmrc/r2/selfassessmentapi/models/selfemployment/AdjustmentsSpec.scala

```diff
@@ -54,8 +54,8 @@ class AdjustmentsSpec extends JsonSpec {
       validateAmount(se, "/includedNonTaxableProfits")
     }
 
-    "reject includedNonTaxableProfits more than 99999999999999.98" in {
-      val se = Adjustments(includedNonTaxableProfits = Some(BigDecimal("99999999999999.99")))
+    "reject includedNonTaxableProfits more than 99999999999.99" in {
+      val se = Adjustments(includedNonTaxableProfits = Some(BigDecimal("999999999999.99")))
```

Comment:
999999999999.99 has 12 digits before the decimal so is **much** larger than the limit. The following (using 99999999999.99+0.01) is a better test:
```suggestion
      val se = Adjustments(includedNonTaxableProfits = Some(BigDecimal("100000000000")))
```
Similarly for other tests and tests in AllowancesSpec.scala.